File: data_monitor/src/main/java/docker_monitor/DM_app/process/object/Trigger.java
package docker_monitor.DM_app.process.object;

public enum Trigger {
    ABOVE("above"),
    BELOW("below");

    private final String val;

    Trigger(String val) {
        this.val = val;
    }
    @Override
    public String toString() {
        return val;
    }
}

File: data_viewer/data_viewer/Model/Notification/Trigger.cs
using System.ComponentModel.DataAnnotations;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace data_viewer.Model.Notification
{
    [JsonConverter(typeof(StringEnumConverter))] 
    public enum Trigger
    {
        [Display(Name = "Above")]
        ABOVE,
        [Display(Name = "Below")]
        BELOW
    }
}